nltk/translate/rouge_score.py

```diff
@@ -0,0 +1,207 @@
+'''ROUGE score implementation
+Link to paper:
+http://www.aclweb.org/anthology/W04-1013
+'''
+
+from nltk.util import ngrams
+from nltk.tokenize import TreebankWordTokenizer, PunktSentenceTokenizer
+from nltk import skipgrams
+from nltk.translate.util import jacknifing, lcs
+import numpy as np
```

Comment:
If the use of numpy is just to get mean and unique, I think it's unnecessary, please use 

```
mean_of_l = sum(l) / len(l)
unique_of_l = len(set(l)
```

Reply:
Done